omf/__init__.py

```diff
@@ -5,20 +5,24 @@
 from __future__ import unicode_literals
 
 from .base import Project
+
+from omf.fileio.geoh5 import GeoH5Writer
+
 from .data import (ColorArray, ColorData,
                    DateTimeArray, DateTimeColormap, DateTimeData,
                    Legend, MappedData,
                    ScalarArray, ScalarColormap, ScalarData,
                    StringArray, StringData,
                    Vector2Array, Vector2Data,
                    Vector3Array, Vector3Data)
+
 from .lineset import LineSetGeometry, LineSetElement
 from .pointset import PointSetGeometry, PointSetElement
 from .surface import SurfaceGeometry, SurfaceGridGeometry, SurfaceElement
 from .texture import ImageTexture
 from .volume import VolumeGridGeometry, VolumeElement
 
-from .fileio import OMFReader, OMFWriter
+from omf.fileio.fileio import OMFReader, OMFWriter
 
 __version__ = '0.9.3'
```

Comment:
will have to introduce a new version.
geoh5 is a new feature, but version 1 or 2 are already used.
We have to move straight to v3.
Probably like 3.0.0-alpha.1 for now

Reply:
Done